geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/data/stats/usage/QueryStat.scala

```diff
@@ -30,7 +30,13 @@ case class QueryStat(typeName: String,
                      planTime: Long,
                      scanTime: Long,
                      hits:     Long,
-                     deleted:  Boolean = false) extends UsageStat
+                     deleted:  Boolean = false) extends DeletableUsageStat {
+  val storeType = QueryStat.storeType
+}
+
+object QueryStat {
+  val storeType = "Accumulo Vector"
```

Comment:
just for future concerns, would it be ok to name this `vector-accumulo` ? should make parsing, etc easier... could name the raster one `raster-accumulo`